site/docs/hive.md

```diff
@@ -72,6 +72,15 @@ catalog.createTable(tableId, schema, spec, tableProperties);
 
 The table level configuration overrides the global Hadoop configuration.
 
+#### Hive on Tez configuration
+
+To use Tez engine on Hive, Tez needs to be upgraded to >= `0.10.1` to contain the fix [Tez-4248](https://issues.apache.org/jira/browse/TEZ-4248).
+
+!!! Warning
+For Hive `2.3.x`, need to manual build from Tez `branch-0.9` for compatibility issue in Tez `0.10.1`.
```

Comment:
```suggestion
For Hive `2.3.x`, you will need to manually build Tez from the `branch-0.9` branch due to a backwards incompatibility issue with Tez `0.10.1`.
```

Reply:
So if I understand correctly, Hive 2.x only works with Tez 0.9.x and not 0.10.x? And therefore one needs to compile and build a specific version of Tez oneself (from that branch?) and then override the version that comes with Hive 2.3.x?

Reply:
Correct.